### ISSUE TITLE
Make sure the username is passed from Wordpress to Etherpad

### ISSUE THREAD

**Issue by JohnMcLear** [boonebgorges/participad#22]
Etherpad supports ?userName IE http://beta.etherpad.org/p/foo?userName=test

**Comment by boonebgorges** [comment]
I'm doing it at the time of Etherpad user creation: https://github.com/boonebgorges/participad/blob/master/includes/class-participad-user.php#L92 Is that enough?

**Comment by JohnMcLear** [comment]
Hrm it seems like enough but I wasn't sure if it worked or not..  Guess this needs testing and confirming.  Leave it open and if you can create a Label called "Waiting for testing" that we can pass off as low hanging fruit

**Comment by boonebgorges** [comment]
Yeah. It probably isn't a bad idea to do it on every pad load anyway, because I'm passing WP's `display_name`, which is a text field that users can change, so it may not be persistent between sessions.
